eventuate-common-in-memory-database/src/main/java/io/eventuate/common/inmemorydatabase/EventuateInMemoryDataSourceBuilder.java

```diff
@@ -0,0 +1,23 @@
+package io.eventuate.common.inmemorydatabase;
+
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+
+import javax.sql.DataSource;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class EventuateInMemoryDataSourceBuilder {
```

Comment:
This just needs to be a single static method `EventuateInMemoryDataSourceFactory.makeDataSource(Collection<EventuateDatabaseScriptSupplier> scripts)`

Reply:
ok